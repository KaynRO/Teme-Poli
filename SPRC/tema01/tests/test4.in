login kayn
add 1 1 1
del 1
logout
login andrei
read 1
read 2
del 2
load
logout
login kayn
load
read 1
get_stat_all
get_stat 1
logout
