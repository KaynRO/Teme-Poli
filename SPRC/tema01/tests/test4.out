--------------------------------
Session key: 1605611530
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
Logout successfully
--------------------------------
Session key: 1605611530
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
Load command should only follow a login
--------------------------------
Logout successfully
--------------------------------
Session key: 1605611530
--------------------------------
Load successfull
--------------------------------
Data ID: 1
No Values: 3
Values: 3.300000 4.400000 5.500000
--------------------------------
Min: 3.300000, Max: 5.500000, Median: 4.400000
Min: 1.200000, Max: 2.200000, Median: 1.700000
Min: 8.800000, Max: 9.900000, Median: 9.350000
--------------------------------
Min: 3.300000, Max: 5.500000, Median: 4.400000
--------------------------------
Logout successfully
